content/pages/FAQ.rst

```diff
@@ -85,7 +88,9 @@ Where do I park?
 Free parking is widely available on campus on Saturdays, with the
 exception of reserved and metered stalls. There is a large parking lot
 to the north of Kelly Engineering that should have enough room for
-everyone.
+everyone. A detailed map can be found on the `About page`_.
+
+.. _About page: .../pages/about.html
```

Comment:
Thanks! I wasn't a hundred percent on that anyway and then I forgot to check it.